starter-code/scripts/models/article.js

```diff
@@ -46,13 +46,23 @@ Article.fetchAll = function() {
     /* When our data is already in localStorage:
     1. We can process and load it,
     2. Then we can render the index page.  */
+    Article.loadAll(JSON.parse(localStorage.hackerIpsum));
+    articleView.renderIndexPage();
+
   } else {
+    $.getJSON("../../data/hackeripsum.json", function(data) {
+      Article.loadAll(data);
+      var articleData = JSON.stringify(data);
+      localStorage.setItem('hackerIpsum', articleData);
```

Comment:
Good! We can also say `localStorage.hackerIpsum = articleData` to achieve the same result. Both are acceptable.